src/web/handlers/GestureHandler.ts

```diff
@@ -41,11 +41,9 @@ export default abstract class GestureHandler implements IGestureHandler {
   protected shouldResetProgress = false;
   protected pointerType: PointerType = PointerType.MOUSE;
 
-  protected delegate: GestureHandlerDelegate<unknown, IGestureHandler>;
+  protected delegate: GestureHandlerWebDelegate;
```

Comment:
This doesn't really seem like keeping web-specific things out of `GestureHandler` 😅.